app/mailers/project_summary_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectSummaryMailer < ApplicationMailer
+
+  def project_summary(user)
+    @user = user
+    mail(to: user.email, subject: "[Learn IT, Girl] Daily summary of your board", sent: DateTime.now)
```

Comment:
I wanted a subject like this : "Subject: [Learn IT, Girl] Daily summary of your board <dd/mm>", Which has a date in subject. The above implementation is fine or you want me to change ?